crates/flipperzero/src/furi/thread.rs

```diff
@@ -13,3 +136,132 @@ pub fn sleep(duration: core::time::Duration) {
         }
     }
 }
+
+/// A unique identifier for a running thread.
+#[cfg(feature = "alloc")]
+pub struct ThreadId(sys::FuriThreadId);
```

Comment:
`sys::FuriThreadId` is really an alias for FreeRTOS's `TaskHandle_t`. There might be some time that you need this for talking with the low-level FreeRTOS APIs, so you probably want to use `#[repr(transparent)]` or just make `ThreadId` a type alias.